pyccel/codegen/printing/cwrappercode.py

```diff
@@ -746,7 +746,8 @@ def get_module_exec_function(self, expr, exec_func_name):
             vars_to_wrap = [v.clone(v.name.lower()) for v in orig_vars_to_wrap]
             for v,w in zip(orig_vars_to_wrap,vars_to_wrap):
                 assign = v.get_user_nodes(Assign)[0]
-                w.set_fst(assign.fst)
+                if assign.fst:
+                    w.set_fst(assign.fst)
```

Comment:
Is this a bug fix?

Reply:
I'm not sure. I don't think we ought to have any cases where an assign doesn't have an fst. But they are occasionally generated in the code printer. I couldn't work out where this missing fst arose so I wasn't able to fix it without this if. However with the if we will occasionally have less detailed error messages (related to the wrapper)

Reply:
I see, can you please add a comment in the code?

Reply:
Done 1434e04f